news/2025/2025-03-08-new-featured-artist-seven-lives.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: "New Featured Artist: SEVEN LIVES"
+date: 2025-03-08 22:15:00 +0000
+---
+
+You haven't heard full flavour music if you haven't heard **SEVEN LIVES**.
+
+![](/wiki/shared/news/2025-03-08-new-featured-artist-seven-lives/header.png)
+
+Hailing from Canada, **SEVEN LIVES** has been exploring music production since their early teens, with immediate inclinations towards multi-genre (a.k.a. "full flavour") rhythm game tracks.
+
+Totaling **16** tracks, [**SEVEN LIVES**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/461) is packed with enough awesome music to occupy your next year of mapping. And as per usual, each track is available as a pre-timed `.osz`!
+*Caravan Palace*, [*onumi*](https://osu.ppy.sh/beatmaps/artists/32), *MUST DIE!*, *MARETU*, [*Kikuo*](https://osu.ppy.sh/beatmaps/artists/314), [*Camellia*](https://osu.ppy.sh/beatmaps/artists/31), [t+pazolite](https://osu.ppy.sh/beatmaps/artists/396), and many many more inspire these tunes, you can be guaranteed to find top-notch music.
+
+You can judge for yourself though:
+
+### [SEVEN LIVES - Statues](https://assets.ppy.sh/artists/461/spirals/SEVEN%20LIVES%20-%20Statues.osz)
```

Comment:
where's the usual "check out this beatmap by (mapper name) or see its preview below" line 😦

Reply:
no maps ig

Reply:
There's https://osu.ppy.sh/beatmapsets/2197744#taiko/4650724.